apps/web/src/views/Cakepad/hooks/ifo/useIFOPoolInfo.ts

```diff
@@ -35,7 +35,7 @@ export const useIFOPoolInfoCtx = (): PoolInfo[] => {
   const version = useAtomValue(ifoVersionAtom)
 
   const { data } = useQuery({
-    queryKey: ['ifoPoolInfo', chainId, addresses, latestTxReceipt, version],
+    queryKey: ['ifoPoolInfo', chainId, addresses, latestTxReceipt, version, ifoContract.address],
```

Comment:
ifoContract should be with optional chaining to avoid crash if it is undefined/null